src/main/java/ja/workshop/spring/części/konfiguracja/xml/c_mvc/Samochód.java

```diff
@@ -0,0 +1,48 @@
+package ja.workshop.spring.części.konfiguracja.xml.c_mvc;
+
+import ja.workshop.spring.części.konfiguracja.xml.c_mvc.silnik.Silnik;
+
+/**
+ * Encja samochód, której reprezentacje trzymane są w lokalnej bazie danych {@link RepozytoriumSamochodów}.
+ *
+ * @author Wiktor Rup
+ */
+public class Samochód {
+
+    private long id;
+    private String nazwa;
+    private Silnik silnik;
+
+    public Samochód() {
+    }
+
+    public Samochód(long id, String nazwa, Silnik silnik) {
+        this.id = id;
+        this.nazwa = nazwa;
+        this.silnik = silnik;
+    }
+
+    public Samochód(Silnik silnik) {
+        this.silnik = silnik;
+    }
+
+    public void setSilnik(Silnik silnik) {
+        this.silnik = silnik;
+    }
+
+    /**
+     * Do poprawnego wyświetlenia interesujących nas dancyh w zwracanym modelu <pre>widok.jsp</pre> niezbędne są publiczne metody get.
```

Comment:
Dancyh -> Danych i bez pustej linii między polami a komentarzami do nich. To samo z komentarzem nad `getId`.